ips.sh

```diff
@@ -84,5 +84,6 @@ centos75x64update5="192.168.222.89"
 centos75x64update6="192.168.222.91"
 centos75x64update7="192.168.222.93"
 centos75x64update8="192.168.222.99"
+centos75x64update9="192.168.222.98"
```

Comment:
Can you please share "uname -a"  output for this new VM?

Reply:
[root@centos74x64 ~]# uname -a
Linux centos74x64 3.10.0-862.27.1.el7.x86_64 #1 SMP Fri Dec 14 16:31:10 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux